### PR TITLE
Remove ul and ol from list of optional tags, they are NOT optional

### PR DESCRIPTION
**Pull request by vithar**
Fixes #issue-472.

### CODE REVIEW

File: lib/bemhtml/index.js [hunk 1]
```diff
@@ -118,7 +118,7 @@ BEMHTML.prototype.render = function(context, entity, tag, js, bem, cls, mix,
 
 var OPTIONAL_END_TAGS = {
   // html4 https://html.spec.whatwg.org/multipage/syntax.html#optional-tags
```
Comment (miripiruni) [inline lib/bemhtml/index.js]: I think we should change that link to https://www.w3.org/TR/html4/index/elements.html

Comment (vithar) [inline lib/bemhtml/index.js]: Done

**Comment by miripiruni** [comment]
@vithar released as bem-xjst@8.7.1

**Comment by miripiruni** [comment]
@vithar also released as bem-xjst@7.7.5

**Pull request merged**